astropy/wcs/src/util.c

```diff
@@ -20,18 +20,28 @@ void set_invalid_to_nan(
   const int* s = stat;
   const int* s_end = stat + ncoord;
   double n;
+  int bit;
```

Comment:
You can do the declaration in the loop (as long as it's at the beginning of the loop block).

Reply:
Done